chapters/01_maj_pattern_book.md

```diff
@@ -0,0 +1,62 @@
+# About this document 
+
+
+##  Welcome 
+
+
+All jugglers are welcome to play with, use, distribute, extend, and otherwise enjoy all 
+the club-passing patterns described in *Madison Area Jugglers' Pattern Book, Version \majpatternbookversion ,* which is licensed by its creators, the Madison Area Jugglers, under the [Creative Commons Attribution 3.0 Unported License](http://creativecommons.org/licenses/by/3.0/deed.en_US) ![Creative Commons License](http://i.creativecommons.org/l/by/3.0/88x31.png){width=7%}. 
```

Comment:
purpose of \majpatternbookversion to help reduce "hunting down" outdated version numbers to update them.  It's a little finicky though, may not be worth it.  Totally fine with removing this if you find it unnecessary.